tools/pylampo-client/client.py

```diff
@@ -0,0 +1,53 @@
+import json
+import socket
+import os
+
+
+class LampoClient:
+    """
+    A simple Lampo client that communicates via a Unix socket.
+    """
+
+    def __init__(self, socket_path: str = None):
+        """
+        Initializes the LampoClient instance.
+
+        Args:
+          socket_path: (Optional) The path to the Lampo socket.
+                      Defaults to '<home_dir>/.lampo/regtest/lampod.socket'.
+        """
+        if socket_path:
+            self.socket_path = socket_path
+        else:
+            home_dir = os.environ["HOME"]
+            self.socket_path = f"{home_dir}/.lampo/regtest/lampod.socket"
```

Comment:
we can have the `socket_path` required inside the code? this default one do not make really sense because the network is fixed

Reply:
No problem
I thought it may make life easier for testing since we will use it with lnprototest tests

Reply:
yes it is bug in test we are pointing to a tmp directory so the home is not really used in the testing one

Reply:
Good point